sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_management_client_async.py

```diff
@@ -0,0 +1,853 @@
+        if not entry.content:
+            raise ResourceNotFoundError("Queue {} does not exist".format(queue_name))
+        runtime_info = QueueRuntimeInfo._from_internal_entity(
+            entry.content.queue_description)
+        runtime_info.name = queue_name
+        return runtime_info
+
+    async def create_queue(self, queue: Union[str, QueueDescription], **kwargs) -> QueueDescription:
+        """Create a queue.
+
+        :param queue: The queue name or a `QueueDescription` instance. When it's a str, it will be the name
+         of the created queue. Other properties of the created queue will have default values decided by the
+         ServiceBus. Use a `QueueDescription` if you want to set queue properties other than the queue name.
+        :type queue: Union[str, ~azure.servicebus.management.QueueDescription]
+        :rtype: ~azure.servicebus.management.QueueDescription
+        """
+        try:
+            queue_name = queue.name  # type: ignore
+            to_create = queue._to_internal_entity()  # type: ignore
+        except AttributeError:
+            queue_name = queue  # type: ignore
+            to_create = InternalQueueDescription()  # Use an empty queue description.
+
+        create_entity_body = CreateQueueBody(
+            content=CreateQueueBodyContent(
+                queue_description=to_create,  # type: ignore
+            )
+        )
+        request_body = create_entity_body.serialize(is_xml=True)
+        with _handle_response_error():
+            entry_ele = cast(
+                ElementTree,
+                await self._impl.entity.put(
+                    queue_name,  # type: ignore
+                    request_body, api_version=constants.API_VERSION, **kwargs)
+            )
+
+        entry = QueueDescriptionEntry.deserialize(entry_ele)
+        result = QueueDescription._from_internal_entity(
+            entry.content.queue_description)
+        result.name = queue_name
+        return result
+
+    async def update_queue(self, queue_description: QueueDescription, **kwargs) -> None:
+        """Update a queue.
+
+        :param queue_description: The properties of this `QueueDescription` will be applied to the queue in
+         ServiceBus. Only a portion of properties can be updated.
+         Refer to https://docs.microsoft.com/en-us/rest/api/servicebus/update-queue.
+        :type queue_description: ~azure.servicebus.management.QueueDescription
+        :keyword timedelta default_message_time_to_live: The value you want to update to.
+        :keyword timedelta lock_duration: The value you want to update to.
+        :keyword bool dead_lettering_on_message_expiration: The value you want to update to.
+        :keyword timedelta duplicate_detection_history_time_window: The value you want to update to.
+        :keyword int max_delivery_count: The value you want to update to.
+        :rtype: ~azure.servicebus.management.QueueDescription
```

Comment:
this line should be removed -- ":rtype: ~azure.servicebus.management.QueueDescription", I guess there're other places needed to be removed as well.

Reply:
Good catch. Interestingly only queue had it. Removed.